keyboards/gergo/keymaps/pcoves/rules.mk

```diff
@@ -0,0 +1,42 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+BALLER          = no # Enable to ball out
+BALLSTEP        = 20 # Multiple in px to move, multiplied by layer number
+SCROLLSTEP      = 1  # Lines to scroll with ball
+MOUSEKEY_ENABLE = no # Mouse keys(+4700), needed for baller
+
+#Debug options
+VERBOSE                = no
+DEBUG_MATRIX_SCAN_RATE = no
+DEBUG_BALLER           = no
+DEBUG_MATRIX           = no
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```

Comment:
You don't actually need the matrix.c added here.  An d I don't think that you need the i2c_master.c file added either.

Reply:
You'r right.
Done